packages/editor/src/components/post-title/index.native.js

```diff
@@ -8,9 +8,15 @@ import { withDispatch } from '@wordpress/data';
 import { withFocusOutside } from '@wordpress/components';
 import { withInstanceId, compose } from '@wordpress/compose';
 
+import { View } from 'react-native';
+
+import styles from './style.scss';
+
 const minHeight = 30;
 
 class PostTitle extends Component {
+	titleViewRef: Object;
```

Comment:
@pinarol gutenberg doesn't support this syntax, we should remove this line if we want to merge back to master.